RecoPPS/Local/test/totemT2NewDigi_reco_cfg.py

```diff
@@ -32,13 +32,13 @@
 process.load('CalibPPS.ESProducers.totemT2DAQMapping_cff')
 process.load('EventFilter.CTPPSRawToDigi.totemT2Digis_cfi')
 process.totemT2Digis.rawDataTag = cms.InputTag("rawDataCollector")
-process.totemDAQMappingESSourceXML.verbosity = cms.untracked.uint32(1)
-process.totemT2Digis.RawUnpacking.verbosity = cms.untracked.uint32(1)
-process.totemT2Digis.RawToDigi.verbosity = cms.untracked.uint32(3)
-process.totemT2Digis.RawToDigi.useOlderT2TestFile = cms.untracked.uint32(1)
-process.totemT2Digis.RawToDigi.printUnknownFrameSummary = cms.untracked.uint32(3)
-process.totemT2Digis.RawToDigi.printErrorSummary = cms.untracked.uint32(3)
-process.totemDAQMappingESSourceXML.multipleChannelsPerPayload = cms.untracked.bool(True)
+process.totemDAQMappingESSourceXML.verbosity = 1
```

Comment:
@oljemark are you sure this is the only config file that needs to be adjusted this way ?

Reply:
Hi, @grzanka . I think this kind of syntax improvement was a nice-to-have, while the ask-fors were making the olderT2 parameter tracked & that one & two print variables into bools. Doing code search for these three parameters they were all changed everywhere & reading the other T2 config files in this directory, I don't see much similar variable-changing statements (and skipping the multiyear old other config files that I don't know are in use anymore).

Reply:
In my opinion its also nice to have, I just would like to be sure that the replacement was done in all places where needed